tools/ci_build/github/azure-pipelines/c-api-noopenmp-packaging-pipelines.yml

```diff
@@ -114,15 +113,15 @@ stages:
 
 - template: stages/java-cuda-packaging-stage.yml
   parameters:
-    CudaVersion: 11.8
+    CudaVersion: 12.2
     SpecificArtifact: ${{ parameters.SpecificArtifact }}
     BuildId: ${{ parameters.BuildId }}
 
 - template: stages/nuget-combine-cuda-stage.yml
   parameters:
       DoCompliance: ${{ parameters.DoCompliance }}
-      CudaVersion: 11.8
-      docker_base_image: 'nvidia/cuda:11.8.0-cudnn8-devel-ubi8'
+      CudaVersion: 12.2
+      docker_base_image: 'nvidia/cuda:12.2.2-cudnn8-devel-ubi8'
```

Comment:
Please use cudnn 9 with cuda 12.

Reply:
Cudnn9 needs cuda 12.3.2 there are many other place need to be changes as well.